### ISSUE TITLE
Position Dot(1037) in Sgaw Karen Language on Noto Sans Myanmar Fonts

### ISSUE THREAD

**Issue by TServicmm** [googlefonts/noto-fonts#1829]
### Font 
NotoSansMyanmar-Regular.ttf
NotoSerifMyanmar-Regular.ttf
NotoSerifMyanmarUI-Regular.ttf

  > Site: https://github.com/googlefonts/noto-fonts/blob/115d38430d957d38307457c036302b7bdbe0bbc4/phaseIII_only/unhinted/otf/NotoSansMyanmar/NotoSansMyanmar-Regular.otf

### Font Version
  > NotoSansMyanmar-Regular.ttf (V2.001)
  > NotoSerifMyanmar-Regular.ttf (V2.001)
  > NotoSerifMyanmarUI-Regular.ttf (V2.000)

### Issue
  > Position left Dot in Sgaw Karen language is still not correct.
  > According from [this](https://github.com/googlefonts/noto-fonts/issues/516) forum, the error is fixed. However, when I test on Word .docx it still wrong position right side.
@kmansourMT Wrote,
![image](https://user-images.githubusercontent.com/69840540/91123021-cff15e00-e6c1-11ea-8861-6b40f45e4518.png)

### Character data
  > [Hex output:](http://unicode.org/charts/PDF/U1000.pdf)
ရ့ = 101b 1037 
ချ့ = 1001 103b 1037 
ၡ့ = 1061 1037
ဘျ့ = 1018 103B 1037
ပျ့ = 1015 103B 1037
ပှ့ၤ = 1015 103E 1037 1064
ပွ့ = 1015 103D 1037
ကွ့ၢ် = 1000 103D 1037 1062 103A   

### Problem Screenshot
  > Test with Noto Sans Myanmar
![image](https://user-images.githubusercontent.com/69840540/91122912-8739a500-e6c1-11ea-9b3e-bcd95b1250ef.png)

### Correct Font View Screen Shot 
 > Test with Karen Text
![image](https://user-images.githubusercontent.com/69840540/91122905-8143c400-e6c1-11ea-8686-e123f6a1ab28.png)

### Important Note 
> For Unicode Number, 1037 = ့ 
We use dot(1037) right side in Myanmar language . 
Also for Karen same dot(1037) should be left side.
Therefore, I assume we can't use that 1037 Unicode number on Noto Sans Myanmar fonts both for Myanmar and Karen languages. 

### Options:
>1. Can we add another Unicode number for Karen left side dot in Noto Sans Myanmar font?
>2. Can we make another Google font name for Karen Sgaw language?

Thank to you again.

**Comment by marekjez86** [comment]
ISO 639-3 language code is ksw

**Comment by marekjez86** [comment]
@TServicmm : thank you for the feedback and for providing hex character data. We do not need a new Unicode codepoint for Karen. we still can use 1037. However, the "code inside the font" needs to be sensitive when language is Karen ( ISO 639-3 language code is ksw ) and then position the dot on the left. We'll test it and make it right (although it may take some time to make it right). thanks again.

**Comment by TServicmm** [comment]
Thank you for additional technical explanation, @marekjex86 
I understand it will take some time to correct.
Same time, I notice another combination  problem too.

### Hex Output
>  ၡူ = 1061 1030 


### Test With Karen Text Font
![image](https://user-images.githubusercontent.com/69840540/91254813-5329b700-e789-11ea-9dc9-eaf5dd7d137d.png)

### Test With Noto Sans Myanmar Font
![image](https://user-images.githubusercontent.com/69840540/91254844-66d51d80-e789-11ea-922a-595ee40cf79d.png)


Many Thanks.

**Comment by nizarsq** [comment]
@TServicmm is this issue reproducible only on Windows? did you test it on Mac and chrome browser using language code ksw?

<img width="877" alt="Screen Shot 2020-08-30 at 3 24 19 PM" src="https://user-images.githubusercontent.com/22158882/91670713-e12cd700-ead4-11ea-9ffd-8da005c3f0b7.png">
<img width="914" alt="Screen Shot 2020-08-30 at 3 24 51 PM" src="https://user-images.githubusercontent.com/22158882/91670721-f570d400-ead4-11ea-9389-6da32c2e845c.png">

**Comment by TServicmm** [comment]
@nizarsq Thank you for your suggestion.  
I just have window computer. 

For my understanding the typing and translation should finish on word document .docx using with Noto Sans font.
After that, the user upload to their website.  If not, why I am kind suggest font issue. :)

Most companies, they embeded(font) the website for to see the text correctly by other user.
Then, the user open Chrome to use their website.

Thank you so much for your help.

**Comment by ohbendy** [comment]
> ၡူ = 1061 1030

This particular combination is not listed in the S'gaw Karen resources I have. Would you prefer the top or bottom version? 
![Screenshot 2021-01-01 at 20 22 17](https://user-images.githubusercontent.com/12471463/103445664-26256d80-4c6f-11eb-820c-ceddf5796463.png)

**Comment by kmansourMT** [comment]
Hi Ben,
To decide this case, I think we need an expert in S’gaw Karen writing conventions.
Best wishes,

Kamal

**Comment by ohbendy** [comment]
Yes, I was hoping the OP was still watching this thread. I've tried to find other experts but they seem a bit scarce!
